Polyfills/Console/Source/Console.cpp

```diff
@@ -16,32 +16,27 @@ namespace
 
 namespace Babylon::Polyfills::Internal
 {
+    static constexpr auto JS_CONSTRUCTOR_NAME = "console";
+
     void Console::CreateInstance(Napi::Env env, Babylon::Polyfills::Console::CallbackT callback)
     {
         Napi::HandleScope scope{env};
 
         Napi::Function func = ParentT::DefineClass(
             env,
-            "Console",
+            JS_CONSTRUCTOR_NAME,
             {
-                ParentT::InstanceMethod("log", &Console::Log),
-                ParentT::InstanceMethod("warn", &Console::Warn),
-                ParentT::InstanceMethod("error", &Console::Error),
+                ParentT::StaticMethod("log", &Console::Log),
+                ParentT::StaticMethod("warn", &Console::Warn),
+                ParentT::StaticMethod("error", &Console::Error),
             });
 
-        auto console = func.New({});
-        Console::Unwrap(console)->m_callback = std::move(callback);
-        const auto existingConsole = env.Global().Get(JS_INSTANCE_NAME);
-        if (!existingConsole.IsUndefined())
-        {
-            Console::Unwrap(console)->m_engineConsole = std::make_unique<Napi::ObjectReference>(Napi::Persistent(existingConsole.As<Napi::Object>()));
-        }
-        env.Global().Set(JS_INSTANCE_NAME, console);
```

Comment:
Why did the `existingConsole` logic get removed?

Also note that this makes `JS_INSTANCE_NAME` in Console.h unused.

Reply:
JS_INSTANCE_NAME removed. Changed from an instance to static def.

Reply:
What about `existingConsole`? This is here because JSC automatically provides a `console` object.